Docs are failing to be published

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
The docs are failing to be published since yesterday. See the [push-build history](https://github.com/rook/rook/actions/workflows/push-build.yaml). The [push-image-to-container-registry action is failing](https://github.com/rook/rook/actions/runs/6745656987/job/18337888281) with the error:

```
if ! git remote show docs > /dev/null 2>&1; then git remote add docs https://***@github.com/rook/rook.github.io.git; fi
# If we are running in GitHub actions, remove the extraheader so our custom GitHub API token is used
if test "$GITHUB_ACTIONS" = "true"; then git config --unset 'http.https://github.com/.extraheader'; fi
git fetch -u docs gh-pages
From https://github.com/rook/rook.github.io
 * branch                gh-pages   -> FETCH_HEAD
 * [new branch]          gh-pages   -> docs/gh-pages
# Switch to root of repo and then run the build and deploy of the documentation
cd /home/runner/work/rook/rook && mike deploy --remote docs --push --rebase --branch gh-pages --update-aliases --prefix docs/rook latest 
usage: mike [-h] [--version] [--debug] COMMAND ...
mike: error: unrecognized arguments: --rebase --prefix
make: *** [Makefile:140: publish.docs] Error 2
make: Leaving directory '/home/runner/work/rook/rook/build/release'
Error: Process completed with exit code 2.
```

**Expected behavior:**
The docs should be successfully published

--- a/build/release/Makefile
+++ b/build/release/Makefile
@@ -137,7 +137,7 @@ publish.docs:
 	if test "$$GITHUB_ACTIONS" = "true"; then git config --unset 'http.https://github.com/.extraheader'; fi
 	git fetch -u docs gh-pages
 	# Switch to root of repo and then run the build and deploy of the documentation
-	cd $(ROOT_DIR) && mike deploy --remote docs --push --rebase --branch gh-pages --update-aliases --prefix $(DOCS_PREFIX) $(DOCS_VERSION) $(DOCS_VERSION_ALIAS)
+	cd $(ROOT_DIR) && mike deploy --remote docs --push --branch gh-pages --update-aliases --deploy-prefix $(DOCS_PREFIX) $(DOCS_VERSION) $(DOCS_VERSION_ALIAS)
 
 # ====================================================================================
 # helm

